src/types/circle.ts

```diff
@@ -172,6 +172,11 @@ export default /* GraphQL */ `
     targetId: ID!
   }
 
+  input CircleSubscriptionInput {
+    "Unique ID."
+    id: ID!
```

Comment:
missing `password: String` for further subscriptions?

Reply:
related discussion: https://mattersnews.slack.com/archives/C01CYECRYTH/p1609814350006600

Reply:
Added. Thanks.